src/NextFederationPlugin.js

```diff
@@ -368,13 +368,13 @@ function generateRemoteTemplate(url, global) {
   }).then(function () {
     const proxy = {
       get: ${global}.get,
-      init: (args) => {
+      init: (...args) => {
```

Comment:
Can you make this ES5 function(){} instead of arrow (sorry this was my bad)

Reply:
All set